docs/Examples/DataAsJSONArray.md

```diff
@@ -0,0 +1,47 @@
+Send data as JSON array
+=======================
+
+Moya is using dictionary as a root container for JSON data. And you shoud
+be using this too. But sometimes you will need to send JSON array as a
```

Comment:
"And you should be using this too." gives me that weird feeling. What do you think about just deleting this line? Doesn't really change the context and feels "nicer" (at least for me 😬)